web/src/search.js

```diff
@@ -90,7 +90,8 @@ export function initialize() {
             search_map = suggestions.lookup_table;
             return suggestions.strings;
         },
-        fixed: true,
+        container: "<div id='search_typeahead' class='typeahead dropdown-menu'></div>",
```

Comment:
I'm not sure about this but just food for thought: It may be better to not override the default container (and hence not duplicate class names), and for the css, scope it to the `typeahead dropdown-menu` inside it's `parentElement`. What do you think?

Reply:
Yeah that seems reasonable :)